### PR TITLE
Add media information to export command 

### PR DESCRIPTION
**Pull request by dklimpel**
### Pull Request Checklist
Follows:
- #14894
- #14969
- #15095

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
  - Feel free to credit yourself, by adding a sentence "Contributed by @github_username." or "Contributed by [Your Name]." to the end of the entry.
* [x] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
* [x] [Code style](https://matrix-org.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

Signed-off-by: Dirk Klimpel [5740567+dklimpel@users.noreply.github.com](mailto:5740567+dklimpel@users.noreply.github.com)

### CODE REVIEW

File: docs/usage/administration/admin_faq.md [hunk 1]
```diff
@@ -70,13 +70,54 @@ output-directory
...
+│   ├───devices
+│   └───profile
+└───media_ids
+    └───<media_id>
+```
+
+The `media_ids` folder contains only the metadata of the media, not the media itself.
+Furthermore, only the `media_ids` that Synapse manages itself are exported.
+If another media repository (eg. [matrix-media-repo](https://github.com/turt2live/matrix-media-repo))
+is used, the data must be exported at this one.
```
Comment (clokep) [inline docs/usage/administration/admin_faq.md:87]:
```suggestion
The `media_ids` folder contains only the metadata of the media, not the media itself.
Furthermore, only the `media_ids` that Synapse manages itself are exported.
If another media repository (e.g. [matrix-media-repo](https://github.com/turt2live/matrix-media-repo))
is used, the data must be exported separately.
```

File: docs/usage/administration/admin_faq.md [hunk 2]
```diff
@@ -70,13 +70,54 @@ output-directory
...
+│   │   ├───global
+│   │   └───<room_id>
+│   ├───connections
+│   ├───devices
+│   └───profile
+└───media_ids
+    └───<media_id>
+```
+
+The `media_ids` folder contains only the metadata of the media, not the media itself.
```
Comment (clokep) [inline docs/usage/administration/admin_faq.md:84]: I think this needs to say that it is the data the user uploaded. (Which I'm not even sure is correct -- is that what this is?)

Comment (dklimpel) [reply docs/usage/administration/admin_faq.md:84]: Yes, this is from media uploaded by this user.

File: docs/usage/administration/admin_faq.md [hunk 3]
```diff
@@ -70,13 +70,55 @@ output-directory
...
+
+for file in $source_directory/*; do
+    filename=$(basename ${file})
+    url=$repository_url/_matrix/media/v3/download/$serverName/$filename
+    echo "Downloading $filename - $url"
+    if ! wget -o /dev/null -P $target_directory $url; then
+        echo "Could not download $filename"
+    fi
+done
 ```
```
Comment (clokep) [inline docs/usage/administration/admin_faq.md:122]: Sounds like this might make another good command! 👍

**Review by clokep — APPROVED** [review]

**Pull request merged**